### PR TITLE
Add new Event Grid event type for SignalR Service

### PR DESCRIPTION
**Pull request by zackliu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by veronicagg — CHANGES_REQUESTED** [review]
@zackliu how are you consuming these definitions from the other specs?

File: specification/eventgrid/data-plane/Microsoft.SignalRService/stable/2018-01-01/SignalRService.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    "paths": {},
+    "definitions": {
+        "SignalRServiceClientConnectionConnectedEventData": {
+            "description": "Schema of the Data property of an EventGridEvent for a Microsoft.SignalRService.ClientConnectionConnected event.",
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "description": "The time at which the event occurred.",
+                    "type": "date-time"
```
Comment (veronicagg) [inline specification/eventgrid/data-plane/Microsoft.SignalRService/stable/2018-01-01/SignalRService.json]: please take a look at type and format for "date-time" https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types

**Review by veronicagg — CHANGES_REQUESTED** [review]
Please check inline comment to resolve semantic validation failure.

**Comment by zackliu** [comment]
@veronicagg We use C# sdk to publish events from our SignalR service runtime

**Review by veronicagg — APPROVED** [review]

**Pull request merged**